sdk/src/logs/batch_log_processor.cc

```diff
@@ -23,7 +23,11 @@ BatchLogProcessor::BatchLogProcessor(std::unique_ptr<LogExporter> &&exporter,
       max_queue_size_(max_queue_size),
       scheduled_delay_millis_(scheduled_delay_millis),
       max_export_batch_size_(max_export_batch_size),
+#  ifdef ENABLE_ASYNC_EXPORT
```

Comment:
`is_export_async` will not be declared without `ENABLE_ASYNC_EXPORT` in `BatchSpanProcessorOptions`. I think we should also remove `is_export_async_` here.

Reply:
Done